index.html

```diff
@@ -42,7 +42,7 @@ <h1>I’m Alexandra.<br>Glad to see you!</h1>
             <!-- the first card............... -->
             <section id="work-section">
                 <div class="cards">
-                    <img src="img2/Snapshoot Portfolio.png" alt="portfolio img">
+                    <img class="card-img"  src="img2/Snapshoot Portfolio.png" alt="portfolio img">
                     <div class="align-text">
                     <h2 class="h2-size">Tonic</h2>
                     <ul class="first-list">
```

Comment:
- [ ] You've done an excellent job with your work cards! I do have a small suggestion to help bring your project even closer to the given [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=9T0c1LaQy7Q424QS-0), though. Specifically, it might be helpful to reduce the height of your cards slightly. 🔍
Currently, your cards are set to a height of `54vh`, which may be causing the images to stretch and look slightly different from the template. By reducing the height by just a bit, you can help ensure that your images display as intended and maintain their proper aspect ratios. 😉

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236501369-b1aad8a7-91fb-4241-9c6c-ce984750b509.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236501496-6fcecefa-72c2-4c31-a9e3-9905d56f7224.png)